miio/huizuo.py

```diff
@@ -147,7 +296,288 @@ def set_brightness(self, level):
+
+    @command(
+        default_output=format_output("Heater powering on"),
+    )
+    def heater_on(self):
+        """Power heater on (only for models with heater)."""
+        if self.model in MODELS_WITH_HEATER:
+            return self.set_property("heater_power", True)
+        else:
+            raise HuizuoException("Your device doesn't support a heater management")
+
+    @command(
+        default_output=format_output("Heater powering off"),
+    )
+    def heater_off(self):
+        """Power heater off (only for models with heater)."""
+        if self.model in MODELS_WITH_HEATER:
+            return self.set_property("heater_power", False)
+        else:
+            raise HuizuoException("Your device doesn't support a heater management")
+
+    @command(
+        click.argument("heat_level", type=int),
+        default_output=format_output("Setting heat level to {heat_level}"),
+    )
+    def set_heat_level(self, heat_level):
+        """Set heat level (only for models with heater)"""
+        if self.model in MODELS_WITH_HEATER:
+            if heat_level not in [1, 2, 3]:
+                raise HuizuoException("Invalid heat level: %s" % heat_level)
+
+            return self.set_property("heat_level", heat_level)
+        else:
+            raise HuizuoException("Your device doesn't support a heat management")
+
+    @command(
+        default_output=format_output(
+            "\n",
+            "------------ Lamp parameters -----------\n"
+            "Power: {result.is_on}\n"
+            "Brightness: {result.brightness}\n"
+            "Color Temperature: {result.color_temp}\n"
+            "\n"
+            "---------- Heater parameters -----------\n"
+            "Heater power: {result.is_heater_on}\n"
+            "Heat level: {result.heat_level}\n"
+            "Heat fault code (0 means 'OK'): {result.heater_fault_code}\n",
+        ),
+    )
+    def status(self) -> HuizuoStatus:
+        """Retrieve properties."""
+
+        return HuizuoStatus(
+            {
+                prop["did"]: prop["value"] if prop["code"] == 0 else None
+                for prop in self.get_properties_for_mapping()
+            }
+        )
+
+
+class HuizuoLampScene(Huizuo):
+    """Support for Huizuo Lamps with additional scene commands"""
+
+    # The next section contains the scene management commands
+    # Right now I have no devices with the scenes for live testing, so the following section
+    # generated based on device specitifations
+
+    @command(
+        default_output=format_output("On/Off switch"),
+    )
+    def scene_on_off(self):
+        """Switch the on/off (only for models with scenes support)."""
+        if self.model in MODELS_WITH_SCENES:
+            return self.set_property("on_off", 0)
+        else:
+            raise HuizuoException("Your device doesn't support scenes")
+
+    @command(
+        default_output=format_output("Add the brightness"),
+    )
+    def brightness_add(self):
```

Comment:
```suggestion
    def brightness_increase(self):
```
Maybe? And `brightness_decrease` for the reverse?

Reply:
Changed to *_ increase/*_ decrease everywhere in code